data/DevPartner.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Konstantin Ivinsky (dev-partner.biz)",
```

Comment:
Hi Konstantin,
I think you should either write your name or dev-partner.biz

Reply:
Hi Aparna,
I created a new file without dev-partner.biz